### PR TITLE
refactored code, removed forced removal of setup script,better banner…

### PR DESCRIPTION
**Pull request by laraib07**
First of all, thanks for this tool.

In this pull request, I changed banner implementation, now it can easily be changed in future. I also changed dependency function to double check dependency. I also removed 'rm -rf $0' so the users can decide themselve if they want to keep or delete the setup script.

Let me know if this is something you want to merge. I tried to respect the code style, but if there's anything you want to change or tweak, feel free to let me know.

### CODE REVIEW

File: setup [hunk 1]
```diff
@@ -3,82 +3,98 @@
...
-    rm -f $0
 
-    echo -e "$G\n[-] Lock Removed$N"
+    echo -e "$G[-] Lock Removed$N\n"
     exit 0
 
   #else exit
-  else
-    error_exit "Authenticatiin Error"
+else
```
Comment (MrAlpha786) [inline setup]: please fix indentation

File: setup [hunk 2]
```diff
@@ -3,82 +3,98 @@
...
     sed -i '2 a termux-fingerprint -c Exit | grep -q "AUTH_RESULT_SUCCESS"; [ $? -ne 0 ] && exit' $FILE
 
-    echo -e "$G\n[-] Lock Set$N"
+    echo -e "$G[-] Lock Set$N\n"
     exit 0
 
   #else skip setup and exit
-  else
-    error_exit "Authenticatiin Error"
+else
```
Comment (MrAlpha786) [inline setup]: please fix indentation

**Review by MrAlpha786 — CHANGES_REQUESTED** [review]
Everything is looking fine. You deleted some indentations, I am sure it's done by mistake. Fix this, don't create an extra commit, amend changes to this commit. And we are good to go.

File: setup (hunk 1 above)
Comment (laraib07) [reply setup]: Done

**Comment by MrAlpha786** [comment]
There was no need for an extra commit . But I squash merge the changes. So its ok now.

**Pull request merged**